### DISCUSSION TITLE
Welcome to Kontenbase Community Discussions!

### DISCUSSION THREAD

**Discussion by kontenbase-team** [kontenbase/kontenbase#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

If you want to get early access as a tester, you can subscribe and fill the survey by entering your name and email at https://kontenbase.com

<img width="1440" alt="Kontenbase Early Access" src="https://user-images.githubusercontent.com/95329195/148442677-01050982-f182-43ee-943b-4f7fd638027e.png">

**Comment by mhaidarhanif** [comment]
Hello Kontenbase team and community friends,

I'm Haidar. Interested in using and developing Kontenbase as the simplest go to platform to create a backend API.

Nice to meet ya!

**Reply by kontenbase-team** [reply]
Welcome, @mhaidarhanif! 👋

**Reply by radiegtya** [reply]
Btw Haidar is also part of our Remote Team across the world. Thank you for starting the discussion! 😃

**Comment by radiegtya** [comment]
Hello everyone,

I am Ega Radiegtya, I am the Founder of Kontenbase. We hope that Kontenbase Community will grow bigger. 
You can introduce yourself here, and let's have a fun discussion! 😁

Warmest Regards,
Kontenbase Team

**Comment by exneval** [comment]
Hi everyone.

It's exneval, thanks for letting me to become part of the community.
Looking forward to make Kontenbase even greater 😄
